astropy/modeling/tests/test_quantities_evaluation.py

```diff
@@ -107,7 +107,7 @@ def test_evaluate(self):
 
     def test_input_units(self):
 
-        self.model.input_units = {'a': u.deg}
+        self.model._input_units = {'a': u.deg}
```

Comment:
I'm not a fan of using private API here, but it's probably better than redefining the model class every time?

Reply:
yeah that was my thought, it's just a testing hack lol